tests/vlan/test_portchannel_tag.py

```diff
@@ -0,0 +1,330 @@
+
+import pytest
+import ptf.packet as scapy
+import ptf.testutils as testutils
+from ptf.mask import Mask
+from collections import defaultdict
+
+import json
+import itertools
+import logging
+
+from tests.common.fixtures.ptfhost_utils import copy_arp_responder_py       # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses        # lgtm[py/unused-import]
+
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_rand_selected_tor  # lgtm[py/unused-import]
+
+from tests.common.config_reload import config_reload
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('topo_t0-56-po2vlan')
+]
+
+
+@pytest.fixture(scope="module")
+def cfg_facts(duthosts, rand_one_dut_hostname):
```

Comment:
>cfg_facts [](http://example.com/codeflow?start=4&length=9)

This file has large overlap with tests/vlan/test_vlan.py. Can you move test cases there?